everware/spawner.py

```diff
@@ -143,15 +111,22 @@ def options_from_form(self, formdata):
 
     @property
     def repo_url(self):
```

Comment:
`"""Parsed repository URL, removing extra information the user might have submitted"""`